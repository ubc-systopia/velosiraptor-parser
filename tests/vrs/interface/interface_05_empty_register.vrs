segment MyUnit {
  interface(base: foo) {
        reg foo [ 8 ] {}
    };
}