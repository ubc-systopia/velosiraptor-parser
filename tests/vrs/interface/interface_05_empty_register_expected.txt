error: empty interface field block
     --> tests/vrs/interface/interface_05_empty_register.vrs:4:5
      |
    4 |      };
      |      ^^^^ remove the block or fill it in explicitly
